CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Release (2024-XX-XX)
+
+- `sqlserverflex`: [v0.7.0](services/sqlserverflex/CHANGELOG.md#v070-2024-09-25)
+  - **Breaking change**: Field `Item` in `ResetUserResponse` is now of type `SingleUser` (previously was `User`)
```

Comment:
Also worth documenting that `DefaultDatabase` is no longer required in the create user payload

Reply:
In the SDK (and in Go) this has no practical impact, but we can add it for transparency